build.sbt

```diff
@@ -70,12 +47,12 @@ def baseProject(module: String, majorMinorVersion: String) = Project(s"$module-p
     scalaVersion := "2.13.11",
     crossScalaVersions := Seq(scalaVersion.value, "2.12.18"),
     scalacOptions := Seq(
+        "-release:11",
         "-feature",
         "-deprecation",
         "-Xfatal-warnings"
     ),
     libraryDependencies += scalaTest,
-    publishTo := sonatypePublishToBundle.value,
```

Comment:
[`gha-scala-library-release-workflow`](https://github.com/guardian/gha-scala-library-release-workflow) will set `publishTo` to the appropriate value (as it happens, [a local-filesystem staging repository](https://github.com/guardian/gha-scala-library-release-workflow/blob/d6ea7cc3eff7868b5ec59879ff0ebf8c044dd8cb/.github/workflows/reusable-release.yml#L159)), no need to set it here.